src/start.ts

```diff
@@ -35,6 +34,14 @@ export const handler = async (_event: any = {}): Promise<any> => {
   const keeperConfig = loadAirkeeperConfig();
   const config = merge(nodeConfig, keeperConfig);
 
+  const baseLogOptions = node.logger.buildBaseOptions(config, {
+    coordinatorId: node.utils.randomString(8),
+  });
+  node.logger.info(
```

Comment:
This is a minor thing:
You may want to import `logger` itself using the destructured import approach, ala `import {logger} from "@api3/airnode-node";`
This would save a lot of repetition of "node". You can do this with a lot of stuff, like `config`, although that would involve renaming your existing uses of config.

Reply:
Yes, I had started doing that same thing but Andre pointed out that by doing that we would be importing the whole library. So for consistency I just import everything the same way.

https://github.com/api3dao/airkeeper/pull/5#discussion_r773746721

Reply:
ah right, sorry about that, thanks :)